app/integration/runtime/src/main/java/io/syndesis/integration/runtime/IntegrationRuntimeAutoConfiguration.java

```diff
@@ -61,4 +62,10 @@ public RouteBuilder integrationRouteBuilder() {
         // of provided ones, know handlers have priority
         return new IntegrationRouteBuilder(location, handlers);
     }
+
+    @Bean
+    @ConditionalOnProperty(prefix = "syndesis.integration.runtime", name = "capture", matchIfMissing = true)
```

Comment:
This is usually in the form:

    @ConditionalOnProperty(prefix = "syndesis.integration.runtime.capture", name = "enabled", matchIfMissing = true)